### PR TITLE
[OAF-23937] Update new interfaces for Browser, Gallery & update share feature

### PR DESCRIPTION
**Pull request by rleojoseph**
# Description
Describe the changes in this pull request.
Explain your rationale of technical decisions you made (unless discussed before).

## Links
Add links to github/jira issues, design documents and other relevant resources (e.g. previous discussions, platform/tool documentation etc.)

# Checklist
- [ ] I have completed all steps in the [Contribution Checklist](../blob/master.github/CONTRIBUTING.md#checklist)

### CODE REVIEW

File: js-miniapp-sample/src/pages/landing.js [hunk 1]
```diff
@@ -126,129 +131,153 @@ const Landing = (props: LandingProps) => {
...
   }
 
+  function isLoggedIn() {
+    MiniApp.miniappUtils
+      .isLoggedIn()
+      .then((response) => {
+        setLoggedIn(response);
+      })
+      .catch((miniAppError) => {
+        console.log('getDarkMode - Error: ', miniAppError);
```
Comment (GondiTilak) [inline js-miniapp-sample/src/pages/landing.js:141]: I think this log should be 'isLoggedIn - error' please check.

**Review by GondiTilak — APPROVED** [review]
Approved, Have minor comments for sample app code please check.

File: js-miniapp-sample/src/pages/landing.js (hunk 1 above)
Comment (rleojoseph) [reply js-miniapp-sample/src/pages/landing.js]: Done

**Review by GondiTilak — APPROVED** [review]

**Review by Climbatize — APPROVED** [review]
LGTM

**Pull request merged**